tests/cypress/tests/rbac_tests/02-view-application-with-rbacuser.js

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2020 Red Hat, Inc.
 // Copyright Contributors to the Open Cluster Management project
 
-const config = Cypress.env("TEST_CONFIG_EXCLUDE_ARGO");
+const config = Cypress.env("RBAC_CONFIG");;
```

Comment:
view seems to have consistent failures:
![image](https://user-images.githubusercontent.com/26282541/116133418-af3c3680-a69c-11eb-86f0-5bdc51158202.png)